src/terraform-provider-signalform/glide.yaml

```diff
@@ -4,7 +4,7 @@ import:
   subpackages:
   - netrc
 - package: github.com/hashicorp/terraform
-  version: 0.11.7
+  version: 0.10.7
```

Comment:
Is it safe to downgrade the tf version? the last time I tried it was pretty bad.